client/src/components/JobDescription.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { Grid, Typography, Link } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import KeyboardArrowUpOutlinedIcon from "@mui/icons-material/KeyboardArrowUpOutlined";
+import { Link as RouterLink } from "react-router-dom";
+
+const JobDescription = (props) => {
+  const [isExpanded, setIsReadMore] = useState(false);
+  const toggle = () => {
+    setIsReadMore(!isExpanded);
+  };
+  const centered = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    height: "5vh",
+  };
+  const ReadMore = ({ children }) => {
+    return (
```

Comment:
we can leverage arrow function's implicit return here as well. we don't need the return statement if we're not running any commands inside this component before the jsx section